app/next-client-app/components/concepts/add-concept.tsx

```diff
@@ -31,14 +37,19 @@ export default function AddConcept({ id, tableId }: AddConceptProps) {
 
   const handleSubmit = async (conceptCode: number) => {
     try {
+      let fieldObj;
```

Comment:
This is a little confusing to read, wondering if it'd be easier something like

```
const contentType = location === "SR-Values" ? "scanreportvalue" : "scanreportfield";

const tableId = location === "SR-Values" 
    ? (await getScanReportField(parentId)).scan_report_table 
    : parentId;
```
I'd almost be tempted to extract this logic to a method, so the logic is encapsulated and cleaner. like `determineContentType` or something.